dataactvalidator/interfaces/validatorStagingInterface.py

```diff
@@ -1,12 +1,12 @@
 from dataactcore.models.validationInterface import ValidationInterface
-from dataactcore.models.stagingModels import Appropriation, ObjectClassProgramActivity, AwardFinancial, AwardFinancialAssistance
+from dataactcore.models.stagingModels import Appropriation, ObjectClassProgramActivity, AwardFinancial, AwardFinancialAssistance, AwardProcurement
 from dataactcore.utils.responseException import ResponseException
 from dataactcore.utils.statusCode import StatusCode
 
 
 class ValidatorStagingInterface(ValidationInterface):
     """Manages all interaction with the staging tables in the validation database."""
-    MODEL_MAP = {"award":AwardFinancialAssistance,"award_financial":AwardFinancial,"appropriations":Appropriation,"program_activity":ObjectClassProgramActivity}
+    MODEL_MAP = {"award":AwardFinancialAssistance,"award_financial":AwardFinancial,"appropriations":Appropriation,"program_activity":ObjectClassProgramActivity,"award_procurement":AwardProcurement}
```

Comment:
I did some experimenting with this and got this mapping tablenames to models:
```
MODEL_MAP = {model[1].__tablename__:model[1] for model in inspect.getmembers(stagingModels, inspect.isclass)
                 if str(model[1].__module__) == "dataactcore.models.stagingModels"}
```

However, it turns out the labels being used here do not all correspond to the table name, the mismatches are:
award->award_financial_assistance
appropriations->appropriation
program_activity->object_class_program_activity

Changing these in setupJobTrackerDB.py led to quite a few errors, so this would need a broader effort to go through and find everywhere we're specifying one of those file types.